dfx_extension/install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+DFX_CACHE_DIR="$(dfx cache show)"
+dfx cache install
```

Comment:
Do we really need to do that? What does unpacking the cache for this version mean? Why does it need to be forcefully done?